### ISSUE TITLE
create new Flow button

### ISSUE THREAD

**Issue by johnnliu** [johnnliu/Flow-Studio-Issues#14]
Button to create new Flow. Saves me having to use the inferior Flow UI

**Comment by johnnliu** [comment]
I don't really want to do this one - currently we have Duplicate as a way to clone a Flow to start.
I'm also thinking about create new from Template.

**Comment by johnnliu** [comment]
yes.  But you would need to edit it in Flow.  So I wonder what is something more that Flow Studio can add:

* may be we can add snippets - so new Flow (with snippet)  ~ need Snippets support
* need to set up connectors if new Flow needs complex connectors   ~ need connector support
* may be it just clones an existing Flow

**Comment by jakemannion** [comment]
Short term _or_ long term... a Create Flow button would be a nice addition. Although I gotta say, the snippet idea has me intrigued as well...

"Yeah hi, can I get a new Flow with four string variables, two integers and an array? Throw in a couple, conditional statements and a side of fries, too." :)

A button or link in the UI that starts the creation process in a new window would be awesome. I usually do Create a blank Flow but I wonder what is most common.

**Comment by johnnliu** [comment]
not sure about conditionals and side of fries.  But I'm thinking a "new" Flow Button will invoke a wizard that:

1. Drop down of existing Flows to copy (this includes copying the trigger and variables, but nothing else - that's what duplicate is for).

**Comment by johnnliu** [comment]
with the new maker portal wizard - we can now do fairly simple URLs to our favourite starting Flows, I've added a bunch as default.  
![image](https://user-images.githubusercontent.com/1279372/57791431-1e564400-7780-11e9-91c3-e1541e21318a.png)
